### PR TITLE
Adding a very useful article and improving the title of an article

### PR DESCRIPTION
**Pull request by therealdhruv**
=>  Adding a very useful article on “Best tools for DevOps” 

 =>  Changing the title name of an article from “Dev Op Terms” to “10 Essential DevOps terms to know” as it's a more appropriate one 
contributions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -329,11 +329,12 @@ Throughout this list you'll see next to each resource and emoji. Here's what eac
 
 ## DevOps
 
+- :bulb: [Best tools for DevOps](https://kush01.hashnode.dev/best-tools-for-devops)
 - :file_folder: [90DaysOfDevOps](https://github.com/MichaelCade/90DaysOfDevOps)
 - :file_folder: [Azure DevOps](https://azure.microsoft.com/en-us/overview/devops-tutorial/)
 - :green_book: [Agile vs Devops](https://enterprisersproject.com/article/2019/1/agile-vs-devops-whats-difference)
 - :books: [Become a DevOps Engineer - Linkedin Learning](https://www.linkedin.com/learning/paths/become-a-devops-engineer?u=57692769)
-- :bulb: [Dev Op Terms](https://enterprisersproject.com/article/2019/8/devops-terms-10-essential-concepts)
+- :bulb: [10 Essential DevOps terms to know as a beginner](https://enterprisersproject.com/article/2019/8/devops-terms-10-essential-concepts)
```
Comment (shahednasser) [inline README.md:337]: Can you instead change the title to the title of the page it links to?

File: README.md [hunk 2]
```diff
@@ -329,11 +329,12 @@ Throughout this list you'll see next to each resource and emoji. Here's what eac
 
 ## DevOps
 
+- :bulb: [Best tools for DevOps](https://kush01.hashnode.dev/best-tools-for-devops)
```
Comment (shahednasser) [inline README.md:332]: Please move this to the end of the list (as resources should be added to the end of a category's list

File: README.md (hunk 1 above)
Comment (therealdhruv) [reply README.md:337]: sure

File: README.md (hunk 2 above)
Comment (therealdhruv) [reply README.md:332]: cool

**Comment by shahednasser** [comment]
Thanks!

**Pull request merged**